src/Contributors.tsx

```diff
@@ -2,26 +2,27 @@ import React from 'react';
 import { StyleSheet, Text, ScrollView } from 'react-native';
 
 const Contributors = () => {
-    const contributorsList = ['Melisa Im', 'Monica Lam', 'Linden Chiu'];
+  const contributorsList = ['Melisa Im', 'Monica Lam', 'Linden Chiu', 'Shilpa Sivarajan'];
```

Comment:
Please move this change to its own PR; each PR should be for one feature/change.

Reply:
Moved this to a new PR from a new branch shilpa/addContributor